modules/tinymce/README.md

```diff
@@ -60,12 +60,12 @@ TinyMCE is an open source software project and we encourage developers to contri
 
 __Basic Rules__
 
-* Contributed code will be licensed under the LGPL license but not limited to LGPL
+* Contributed code will be licensed under the MIT license
```

Comment:
Do we need the " but not limited to MIT" bit?

Reply:
I'm not sure. There are a few parts of this change that weren't fully specced out.

Reply:
I assume it was added for our commercial users in the case of LGPL so we could re-license it to suite them. So the question would be if they'd be happy with MIT being the only option.

Reply:
I've thought about this and it seems more about making clear to contributors that code we accept may be relicensed and not released as LGPL (i.e. our commercial release).

The MIT license allows commercial use so I think "not limited to MIT" is unnecessary.